circle.html

```diff
@@ -125,6 +125,17 @@
       }
     })
   </script>
+</script>
+<div>
+  <script>
+  function createDownload() {
+      const downloadURL = document.getElementById('myCanvas').toDataURL();
+      console.log(document.getElementById('myCanvas'));
```

Comment:
@aksh-hash you forgot to remove this console log statement.

Reply:
Sure, i'll do it. Thankyou